components/casemap/Cargo.toml

```diff
@@ -23,9 +23,10 @@ all-features = true
 icu_collections = { workspace = true, features = ["alloc"] }
 icu_locale_core = { workspace = true, features = ["alloc"] }
 icu_properties = { workspace = true }
-icu_provider = { workspace = true, features = ["alloc"] }
+icu_provider = { workspace = true }
```

Comment:
Issue: This PR breaks the meaning of the `alloc` feature in the 2.0.x and 2.1.x releases. Changing no-default-features behavior is not forbidden, but given that this is a low-level crate, we should do such a change only if it is better than feasible alternatives.

I think splitting the `alloc` feature into two pieces, as suggested before, would be an example of a feasible alternative. There might be others.